Eevee.Sleep.Bot/Models/InGameAnnouncementDetailModel.cs

```diff
@@ -0,0 +1,19 @@
+using JetBrains.Annotations;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace Eevee.Sleep.Bot.Models;
+
+[BsonIgnoreExtraElements]
+public record InGameAnnouncementDetailModel: InGameAnnouncementMetaModel {
+    [UsedImplicitly]
+    public required string Content { get; init; }
+
+    [UsedImplicitly]
+    public required string ContentHash { get; init; }
+
+    [UsedImplicitly]
+    public required string Updated { get; init; }
```

Comment:
Should be in type of `DateTime` with timezone info (maybe need to check if the announcement changes the date according to local timezone?)
Remember to store it in UTC and suffix this property with `Utc`

Reply:
It's in yyyy-mm-dd format. I don't have time.
I don't want to make errors by converting, so I think it should save the raw string here.